### PR TITLE
MULTIARCH-3683: Add dev flags in destroy cluster powervs command

### PR DESCRIPTION
**Pull request by Neha-dot-Yadav**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:

Fixes https://issues.redhat.com/browse/MULTIARCH-3683

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: cmd/cluster/powervs/destroy.go [hunk 1]
```diff
@@ -35,8 +35,17 @@ func NewDestroyCommand(opts *core.DestroyOptions) *cobra.Command {
 	cmd.Flags().StringVar(&opts.PowerVSPlatform.Region, "region", opts.PowerVSPlatform.Region, "IBM Cloud region. Default is us-south")
 	cmd.Flags().StringVar(&opts.PowerVSPlatform.Zone, "zone", opts.PowerVSPlatform.Zone, "IBM Cloud zone. Default is us-south")
 	cmd.Flags().StringVar(&opts.PowerVSPlatform.VPCRegion, "vpc-region", opts.PowerVSPlatform.VPCRegion, "IBM Cloud VPC Region for VPC resources. Default is us-south")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.VPC, "vpc", "", "IBM Cloud VPC Name")
```
Comment (enxebre) [inline cmd/cluster/powervs/destroy.go:38]: I can't tell what this new flags are for by reading their desc, it'd be good to clarify

Comment (Neha-dot-Yadav) [reply cmd/cluster/powervs/destroy.go]: Thanks, added description of the flags to clarify.

**Pull request merged**